packages/plugin-ext/src/main/browser/tasks-main.ts

```diff
@@ -219,7 +219,7 @@ export class TasksMainImpl implements TasksMain, Disposable {
     }
 
     protected fromTaskConfiguration(task: TaskConfiguration): TaskDto {
-        const { group, presentation, _scope, _source, ...common } = task;
+        const { group, presentation, _scope, _source, taskType, ...common } = task;
```

Comment:
taskType does not seem to be converted, so we can leave it as part of ...comon, right ?